sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 # Release History
 
-## 1.4.0-beta.3 (Unreleased)
+## 1.4.0-beta.3 (2025-03-28)
 
 ### Features Added
 
 * Added support for emitting Application Insights Custom Events.
   ([#48378](https://github.com/Azure/azure-sdk-for-net/pull/48378))
 
+* Added new api `UseAzureMonitorExporter` which enables all three signals.
```

Comment:
Could you please update this in detail, borrow idea from here - https://github.com/open-telemetry/opentelemetry-dotnet/blob/868ae707fca03a0c5dac7f39a11d79dc50bf3bfc/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md?plain=1#L220

Live metrics support could be sub-bullet under this one.

Reply:
done.